java/runtime/src/main/java/io/ray/runtime/placementgroup/PlacementGroupUtils.java

```diff
@@ -95,8 +94,9 @@ private static PlacementGroupImpl generatePlacementGroupFromPbData(
    */
   public static PlacementGroupImpl generatePlacementGroupFromByteArray(
       byte[] placementGroupByteArray) {
-    Preconditions.checkNotNull(
-        placementGroupByteArray, "Can't generate a placement group from empty byte array.");
+    if (placementGroupByteArray == null) {
```

Comment:
Why change this?

Reply:
because the user can judge whether it exists with the name before create it.